server/apps/immich/src/api-v1/asset/asset.controller.ts

```diff
@@ -228,6 +228,11 @@ export class AssetController {
     return this.assetService.getAssetCountByUserId(authUser);
   }
 
+  @Authenticated()
+  @Get('/archived-count-by-user-id')
```

Comment:
Not a huge fan of this url. Can this be "/counts/archived"?

Reply:
Do you mean `/asset/counts/archived` or `/counts/archived` endpoint?

Reply:
I've changed `/asset/archived-count-by-user-id` endpoint to `/asset/stat/archive`. I think it's more generic. Is it fine?